lldb/source/Plugins/Process/scripted/ScriptedProcess.h

```diff
@@ -52,7 +52,7 @@ class ScriptedProcess : public Process {
 
   void DidResume() override;
 
-  Status DoResume() override;
+  Status DoResume(lldb::RunDirection direction) override;
```

Comment:
We should use `DoResumeReverse()` in Process.h and this change won't need to happen.

Reply:
The decision was to use a direction flag.